src/System.IO.FileSystem/tests/Performance/System.IO.FileSystem.Performance.Tests.csproj

```diff
@@ -1,8 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <PropertyGroup>
-    <Configuration Condition="'$(Configuration)'==''">Windows_Debug</Configuration>
```

Comment:
I guess this pattern is useless. There are 4 other test projects with this in, can you remove from the others?